storage/aptosdb/src/lib.rs

```diff
@@ -1893,18 +1893,49 @@ impl DbWriter for AptosDB {
             )?;
 
             // Delete the genesis transaction
-            StateMerklePruner::prune_genesis(self.state_merkle_db.clone(), &mut batch)?;
             LedgerPruner::prune_genesis(
                 self.ledger_db.clone(),
                 self.state_store.clone(),
                 &mut batch,
             )?;
 
+            self.ledger_pruner
+                .pruner()
+                .save_min_readable_version(version, &batch)?;
+
+            let mut state_merkle_batch = SchemaBatch::new();
+            StateMerklePruner::prune_genesis(
+                self.state_merkle_db.clone(),
+                &mut state_merkle_batch,
+            )?;
+
+            self.state_store
+                .state_pruner
+                .pruner()
+                .save_min_readable_version(version, &state_merkle_batch)?;
```

Comment:
There's a lot of individual calls to `save_min_readable_version`. If we do this anywhere else, I'd expect a wrapper/utility function, e.g., `save_min_readable_version_on_all_pruners`?

Reply:
This is the only place :)

We can wrap them later if necessary.